LibreHardwareMonitorLib/Hardware/D3DDisplayDevice.cs

```diff
@@ -1067,34 +52,34 @@ public static bool GetDeviceInfoByName(string displayDeviceName, out D3DDeviceIn
 
             for (uint nodeId = 0; nodeId < nodeCount; nodeId++)
             {
-                D3DKMT_NODEMETADATA nodeMetaData;
-                GetNodeMetaData(out status, adapter, nodeId, out nodeMetaData);
-                if (status != NtStatus.Success) return false;
+                GetNodeMetaData(out status, adapter, nodeId, out D3dkmth.D3DKMT_NODEMETADATA nodeMetaData);
+                if (status != WinNt.STATUS_SUCCESS)
+                    return false;
+
 
-                D3DKMT_QUERYSTATISTICS_NODE_INFORMATION nodeInformation;
-                GetQueryStatisticsNode(out status, adapter, nodeId, out nodeInformation);
-                if (status != NtStatus.Success) return false;
+                GetQueryStatisticsNode(out status, adapter, nodeId, out D3dkmth.D3DKMT_QUERYSTATISTICS_NODE_INFORMATION nodeInformation);
+                if (status != WinNt.STATUS_SUCCESS)
+                    return false;
 
-                deviceInfo.Nodes[nodeId] = new D3DDeviceNodeInfo()
+
+                deviceInfo.Nodes[nodeId] = new D3DDeviceNodeInfo
                 {
-                    Id = nodeId,
-                    Name = GetNodeEngineTypeString(nodeMetaData),
-                    RunningTime = nodeInformation.GlobalInformation.RunningTime.QuadPart,
-                    QueryTime = DateTime.Now
+                    Id = nodeId, Name = GetNodeEngineTypeString(nodeMetaData), RunningTime = nodeInformation.GlobalInformation.RunningTime.QuadPart, QueryTime = DateTime.Now
```

Comment:
I think it was more readable with the newlines

Reply:
Agreed, somehow this got moved to a single line.